src/EditorFeatures/CSharp/EventHookup/EventHookupCommandHandler_TabKeyCommand.cs

```diff
@@ -231,7 +235,13 @@ private void GenerateAndAddEventHandler(ITextView textView, ITextBuffer subjectB
 
             var typeDecl = eventHookupExpression.GetAncestor<TypeDeclarationSyntax>();
 
-            var typeDeclWithMethodAdded = CodeGenerator.AddMethodDeclaration(typeDecl, generatedMethodSymbol, document.Project.Solution.Workspace, new CodeGenerationOptions(afterThisLocation: eventHookupExpression.GetLocation()));
+            var codeGenerator = document.Document.GetRequiredLanguageService<ICodeGenerationService>();
+
+            var codeGenOptions = new CodeGenerationOptions(
+                new CodeGenerationContext(afterThisLocation: eventHookupExpression.GetLocation()),
```

Comment:
thanks for renaming it to 'Context' that helps a lot with "options" confusion :)